redash/tasks.py

```diff
@@ -317,3 +318,47 @@ def execute_query(self, query, data_source_id, metadata):
 @celery.task(base=BaseTask)
 def record_event(event):
     models.Event.record(event)
+
+def notify_hipchat(alert, html, new_state):
+    try:
+        hipchat_client = hipchat.HipChat(token=settings.HIPCHAT_API_TOKEN)
+        message = '[' + new_state.upper() + '] ' + alert.name + '<br />' + html
+        hipchat_client.message_room(settings.HIPCHAT_ROOM_ID, settings.NAME, message, message_format='html')
+    except:
+        logger.exception("hipchat send ERROR.")
+
+def notify_mail(alert, html, new_state, app):
+    recipients = [s.email for s in alert.subscribers()]
+    logger.debug("Notifying: %s", recipients)
+    try:
+        with app.app_context():
+            message = Message(recipients=recipients,
+                              subject="[{1}] {0}".format(alert.name, new_state.upper()),
+                              html=html)
+            mail.send(message)
+    except:
+        logger.exception("mail send ERROR.")
+
+def notify_webhook(alert, query, html, new_state):
+    http_client = httplib2.Http(timeout=30)
```

Comment:
[`requests`](http://docs.python-requests.org/en/latest/) is already in the project dependencies, please use it.